conda_build/os_utils/liefldd.py

```diff
@@ -50,10 +50,14 @@
         # Fallback for lief<0.15.
         LOAD_COMMAND_TYPES = lief.MachO.LOAD_COMMAND_TYPES
     try:
-        ELF_CLASS = lief.ELF.Header.CLASS
+        ELF64 = lief.ELF.Header.CLASS.ELF64
+        ELF32 = lief.ELF.Header.CLASS.ELF32
     except AttributeError:
         # Fallback for lief<0.15.
-        ELF_CLASS = lief.ELF.ELF_CLASS
+        ELF64 = lief.ELF.ELF64
```

Comment:
the `ELF64` name wasn't used in lief prior to 0.15, was it? should this line be
`ELF64 = lief.ELF.ELF_CLASS.CLASS64`   ?

Reply:
Indeed. Copy paste / search replace error :)